frontend/app/templates/components/transportation/tdf/in-out-splits.hbs

```diff
@@ -12,10 +12,10 @@
       </tr>
     </thead>
     <tbody class="ui form">
-      <Transportation::Tdf::InOutSplits::TableRow @data={{data.am}} @label="AM" data-test-in-out-split="am"/>
-      <Transportation::Tdf::InOutSplits::TableRow @data={{data.md}} @label="Midday" />
-      <Transportation::Tdf::InOutSplits::TableRow @data={{data.pm}} @label="PM" />
-      <Transportation::Tdf::InOutSplits::TableRow @data={{data.saturday}} @label="Saturday" />
+      <Transportation::Tdf::InOutSplits::TableRow @data={{data.am}} @model={{model}} @label="AM" data-test-in-out-split="am"/>
+      <Transportation::Tdf::InOutSplits::TableRow @data={{data.md}} @model={{model}} @label="Midday" />
+      <Transportation::Tdf::InOutSplits::TableRow @data={{data.pm}} @model={{model}} @label="PM" />
+      <Transportation::Tdf::InOutSplits::TableRow @data={{data.saturday}} @model={{model}} @label="Saturday" />
```

Comment:
We're now passing a new `model` attribute to this component, but I don't see it defined on the component itself.